.gitignore

```diff
@@ -38,6 +38,7 @@ ipch/
 GeneratedFiles/
 .moc/
 /UI/obs.rc
+*.code-workspace
```

Comment:
I may misunderstand something, but workspace are normally outside of the local repo because they agglomerate various folder.

I don't see the point of adding it to the `.gitignore`.

Reply:
Developer preference catch-all; I keep workspaces self-contained, but I'm a new submitter here, I'll remove it from the ignore.